dataland-frontend/src/components/forms/parts/elements/derived/NaceCodeSelector.vue

```diff
@@ -29,6 +29,7 @@
               :modelValue="selectedTreeNodes.has(slotProps.node.key)"
               @update:modelValue="(isChecked: boolean) => handleNodeCheckboxClick(slotProps.node.key, isChecked)"
               :binary="true"
+              :disabled="disableCheckboxes && slotProps.node.children.length > 0"
```

Comment:
I find it very irritating to have checkboxes there that you can not click. We could try it with v-if instead?
The greyed out boxes only look disabled when someone compares them with the enabled boxes. Users would be confused.